### ISSUE TITLE
When is this package going to be released?

### ISSUE THREAD

**Issue by batmanhit** [bootstrap-vue/bootstrap-vue-next#3]
**What you are trying to do.**

We have upgraded all of our current web application to vue 3 but it heavily depends on `bootstrap-vue` which is getting less compatible with `vue 3` although `@vue/compat` is supposed to fix this.

Is there any estimated release date for this repository?
Cheers.

**Comment by xvaara** [comment]
There are quite a few releases: https://www.npmjs.com/package/bootstrap-vue-next?activeTab=versions 

The repo is now in a weird state, see #2 .
There people (like me) using it in production. I don't think there is a clear plan what constitutes as a stable release at the moment.

**Comment by StirStudios** [comment]
Yes we are hoping this can be solved swiftly as well. Sounds like @VividLemon has been working hard making several improvements, but a repo was deleted.
